### ISSUE TITLE
Alignment point for text should not depend on “script”

### ISSUE THREAD

**Issue by fantasai** [w3c/svgwg#629]
> alignment point
>    The point on a typographic character that should be aligned with the current text position. It is determined by the glyph cell metrics and may depend on the script and inline-base direction. 

> Glyphs are positioned relative to a particular point on each glyph known as the alignment point. For horizontal writing-modes, the glyphs' alignment points are vertically aligned while for vertical writing-modes, they are horizontally aligned. The position of the alignment point depends on the script. For example, Western glyphs are aligned at the bottom of capital letters, northern indic glyphs are aligned at the top of a horizontal stroke near the top of the glyphs, and far-eastern glyphs are aligned either at the bottom or center of the glyph. 

That last sentence in particular is complete nonsense. It would mean that for a text with a given y position, Latin characters would sit on the baseline, CJK characters would be pushed up a bit higher, and indic glyphs would be dropped below the baseline. Luckily that doesn't seem to actually happen in implementations. (Also, how is the spec expecting the UA to determine the "script" of a glyph? Sure letters can belong to a specific writing system, but punctuation doesn't.)

The alignment point in the block axis should be based on the `alignment-baseline` and `dominant-baseline` properties. That's what they're for.

**Comment by AmeliaBR** [comment]
I agree that this wording has conflated the idea of baselines — and the fact that different scripts have different natural baselines — with the alignment point as used in the SVG algorithms, to mean the point where the start edge of the glyph intersects the relevant baseline. 

In SVG 1, the [`dominant-baseline` property](https://www.w3.org/TR/SVG11/text.html) had a `use-script` value which would have created this script-dependent alignment. But that was dropped in CSS Text 3.

The second paragraph you quote seems to have been adapted from [a SVG Tiny 1.2 definition of baseline](https://www.w3.org/TR/SVGTiny12/text.html#TextFonts).  SVG Tiny didn't support the `dominant-baseline` or `alignment-baseline` properties, but the layout algorithm seems to assume an automatic baseline selection based on script.

Since a lot of the SVG Tiny 1.2 details about text layout were copied over into SVG 2, it probably requires a careful review to make sure that similar ambiguous assumptions aren't incorporated elsewhere.
